vault_program/src/burn_withdrawal_ticket.rs

```diff
@@ -76,7 +76,20 @@ pub fn process_burn_withdrawal_ticket(
         fee_amount,
         burn_amount,
         out_amount,
-    } = vault.burn_with_fee(vault_staker_withdrawal_ticket.vrt_amount(), min_amount_out)?;
+    } = vault.burn_with_fee(
+        vault_staker_withdrawal_ticket.vrt_amount(),
+        vault_staker_withdrawal_ticket.min_amount_out(),
+    )?;
+
+    if out_amount < vault_staker_withdrawal_ticket.min_amount_out() {
```

Comment:
i think this check is done with burn with fee? don't need it here actually

Reply:
prolly why that codecov error below - erroring out earlier

<img width="1075" alt="image" src="https://github.com/user-attachments/assets/11242492-e12d-4d77-bb23-ca334fd2b41b">

Reply:
Yep, that tracks, will fix